Assets/LeapMotion/Core/Scripts/Utils/RuntimeColliderGizmos.cs

```diff
@@ -16,10 +16,16 @@ public class RuntimeColliderGizmos : MonoBehaviour, IRuntimeGizmoComponent {
     public Color color = Color.white;
     public bool useWireframe = true;
     public bool traverseHierarchy = true;
+    public bool drawTriggers = false;
+
+    void Start() { }
```

Comment:
Why was the start callback added?

Reply:
to give the component a checkbox so it can be enabled and disabled.

Reply:
Ahh makes sense, might be good to add a comment in that case 👍

Reply:
Bump?

Reply:
woops sorry added